tests/components/config/test_entity_registry.py

```diff
@@ -195,6 +204,7 @@ async def test_update_entity(hass, client):
             "disabled_by": None,
             "entity_category": None,
             "entity_id": "test_domain.world",
+            "hidden_by": RegistryEntryHider.USER,
```

Comment:
Can you pass a string to make sure that works. 

Also, can you test an invalid value.

Reply:
Fixed in `2d8ab0f`